CONTRIBUTING.md

```diff
@@ -121,9 +121,6 @@ This project formats its source code using Prettier. The most enjoyable way to
 use Prettier is to let is format code for you when you save. You can [integrate
 it into your editor][integrate prettier].
 
-If for whatever reason you don't want to do that, you can run
-`npm run prettier` to format the code from the command line.
-
```

Comment:
I think we should replace this with a sentence explaining that we run lint/prettier in a pre commit hook.

Reply:
Good idea. I've added a note. 
And thanks for the `git commit --no-verify`, I didn't know it :-)